.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
+
+[*.js]
+indent_style = space
+indent_size = 4
```

Comment:
Probably worth keeping this consistent with the other files? I have no objections with them being the same really...

Reply:
Could you point out which other extensions as using indent rule globally is a bad idea.

Reply:
https://github.com/dom111/webdav-js/pull/29/commits/4d1a7b51699e125885522056f5ef32b8d8b629ad done